### PR TITLE
Gut blobs

### LINKED ISSUE #5071: Remove version from blobs
#### Problem

A version field was added to blobs in #5057 to support rollbacks. 

#### Proposed Solution

Find a way to support rollbacks without a version field, or conclude that this is the only way. 

Tag: @carllin @mvines @aeyakovenko @pgarg66

### PR DESCRIPTION
**Pull request by mvines**
Blobs were carrying around a bunch of dead code.

Fixes #5071

### CODE REVIEW

**Comment by mvines** [comment]
@sagar-solana - I want to nuke BLOB_DATA_ALIGN too, think that'll cause issues?  It only existed for erasure, which we don't use with Blobs anymore

File: core/src/blob.rs [hunk 1]
```diff
@@ -147,11 +138,7 @@ macro_rules! align {
 }
 
 pub const BLOB_HEADER_SIZE: usize = align!(SIZE_RANGE.end, BLOB_DATA_ALIGN); // make sure data() is safe for erasure
-pub const SIGNABLE_START: usize = PARENT_RANGE.start;
-
-pub const BLOB_FLAG_IS_LAST_IN_SLOT: u32 = 0x2;
-
-pub const BLOB_FLAG_IS_CODING: u32 = 0x1;
+pub const SIGNABLE_START: usize = SIGNATURE_RANGE.end;
```
Comment (sagar-solana) [inline core/src/blob.rs]: We probably don't need them to support signing either. For gossip, the CrdsValue contained within is signed.

**Comment by sagar-solana** [comment]
Yeah I think that's safe to remove. I wanted to see if we could just use Packet for gossip and do away with blobs altogether.

**Comment by mvines** [comment]
⏸ until Sagar sees if we can rip out Blobs entirely

**Comment by mvines** [comment]
My plan worked and @sagar-solana one upped me via #6937

**Pull request closed without merging**